flycheck-clj-kondo.el

```diff
@@ -53,7 +53,7 @@
   your file extension doesn't match your major-mode.")
 
 (defmacro flycheck-clj-kondo--define-checker
-    (name lang mode &rest extra-args)
+    (name lang modes &rest extra-args)
```

Comment:
Isn't this a breaking change?

Reply:
With https://github.com/borkdude/flycheck-clj-kondo/pull/26/commits/368bbb026a7346cde10efdfd60abb1637d5d95b7 it is no longer a breaking change as per the flycheck docs `:modes` can be either a symbol ofter major mode or a list of symbols of major modes.

But you were right, my previous changes to `:predicate` would have been breaking.